compiler/rustc_resolve/src/diagnostics.rs

```diff
@@ -950,9 +950,7 @@ impl<'a> Resolver<'a> {
         self.add_typo_suggestion(err, suggestion, ident.span);
 
         let import_suggestions =
-            self.lookup_import_candidates(ident, Namespace::MacroNS, parent_scope, |res| {
-                matches!(res, Res::Def(DefKind::Macro(MacroKind::Bang), _))
-            });
+            self.lookup_import_candidates(ident, Namespace::MacroNS, parent_scope, is_expected);
```

Comment:
This is gonna have a merge conflict with the other PR, right?

Reply:
That other PR includes this one as the first commit (with identical hash even), so should be fine. ^^